make/testing_flags.mk

```diff
@@ -45,7 +45,7 @@ endif
 ifneq ($(timeout),)
 TEST_FLAGS += -test.timeout=$(timeout)
 else
-TEST_FLAGS += -test.timeout=40m
+TEST_FLAGS += -test.timeout=50m
```

Comment:
Is this because the extra start/stop cycle makes the test run longer?

Reply:
Not locally much in my experience, but it seems that CI builds became slow recently. I'm still investigating why, eg: https://travis-ci.org/github/lightningnetwork/lnd/jobs/740247090

Reply:
Travis has become somewhat slower (also jobs queue up a lot more) since Nov 2 when they announced that they start phasing out unlimited free minutes for OSS builds..

Reply:
With the latest #4655 change this might not be needed anymore?

Reply:
Yeah, I think we can drop this commit. Though Travis is almost green, not sure if we want to risk another round of waiting for it.

Reply:
removed